api/candlepin-api-spec.yaml

```diff
@@ -8391,7 +8391,6 @@ components:
           minLength: 1
         signature:
           type: string
-          minLength: 1
```

Comment:
We should probably remove signature from required section above.

Reply:
That permits nulls and not supplying the field at all. I suppose that's a matter of semantics, but I restored the requirement after checking our design docs. They weren't 100% clear, but leaned more toward "must specify a signature" than "optional."